kustomize/README.md

```diff
@@ -67,18 +67,22 @@ You need to have a Kubernetes cluster where you will deploy the Online Boutique'
 
 Here is the list of the variations available as Kustomize components that you could leverage:
 
-- [**Changing to the Cymbal Shops Branding**](components/cymbal-branding)
+- [**Change to the Cymbal Shops Branding**](components/cymbal-branding)
   - Changes all Online Boutique-related branding to Google Cloud's fictitious company — Cymbal Shops. The code adds/enables an environment variable `CYMBAL_BRANDING` in the `frontend` service.
-- [**Integrating with Google Cloud Operations**](components/google-cloud-operations)
+- [**Integrate with Google Cloud Operations**](components/google-cloud-operations)
   - Enables Monitoring (Stats), Tracing, Profiler, and Debugger for various services within Online Boutique. The code removes the existing environment variables (`DISABLE_STATS`, `DISABLE_TRACING`, `DISABLE_PROFILER`, `DISABLE_DEBUGGER`) from appropriate YAML config files.
-- [**Integrating with Memorystore (redis)**](components/memorystore)
+- [**Integrate with Memorystore (redis)**](components/memorystore)
   - The default Online Boutique deployment uses the in-cluster `redis` database for storing the contents of its shopping cart. The Memorystore deployment variation overrides the default database with its own Memorystore (redis) database. These changes directly affect `cartservice`.
-- [**Securing with Network Policies**](components/network-policies)
+- [**Secure with Network Policies**](components/network-policies)
   - Deploy fine granular `NetworkPolicies` for Online Boutique.
-- [**Creating Kubernetes Service Accounts**](components/service-accounts)
+- [**Create Kubernetes Service Accounts**](components/service-accounts)
```

Comment:
**Rationale for this change:**
To be consistent with cloud.google.com documentation, I wanted to use imperative (e.g., "Create" instead of "Creating") headings.

Reply:
Good catch/point.